apps/charterafrica/src/components/Datasets/DatasetCard.js

```diff
@@ -0,0 +1,107 @@
+import { Box, Grid, Typography, Chip, Button } from "@mui/material";
+import { useState } from "react";
+
+import LineClampedRichTypography from "@/charterafrica/components/LineClampedRichTypography";
+import formatDateTime from "@/charterafrica/utils/formatDate";
+
+function DatasetCard({
+  formats,
+  notes,
+  title,
+  created,
+  readMore,
+  readLess,
+  updated,
+  author,
+  sx,
+}) {
+  const [showAll, setShowAll] = useState(false);
+
+  const handleClick = () => setShowAll(!showAll);
+
+  return (
+    <Box
+      sx={{
+        border: "none",
+        p: 2.5,
+        ...sx,
+      }}
+    >
+      <Grid
+        container
+        spacing={2}
+        justifyContent="space-between"
+        direction={{ xs: "column", md: "row" }}
+      >
+        <Grid item xs={12} md={4}>
+          <LineClampedRichTypography
+            color="neutral.dark"
+            lineClamp={1}
+            order={1}
+            variant="h5SmallSemiBold"
+            sx={{ mb: 1 }}
+          >
+            {title}
+          </LineClampedRichTypography>
+          <Typography variant="p1" color="neutral.main" sx={{ mb: 1 }}>
+            Updated {formatDateTime(updated, { includeTime: false })} | Created{" "}
+            {formatDateTime(created, { includeTime: false })}
+          </Typography>
+          <Typography variant="p1SemiBold" color="neutral.main">
+            {author}
+          </Typography>
+        </Grid>
+        <Grid item xs={12} md={4} order={{ xs: 4, sm: 3 }}>
+          <LineClampedRichTypography
+            color="common.black"
+            lineClamp={showAll ? -1 : 3}
+            order={1}
+            variant="p1"
+            sx={{ mb: 1 }}
+          >
+            {notes}
+          </LineClampedRichTypography>
+          <Button
```

Comment:
1. There is some left margin/padding on this button
2. It appears everywhere even where it should e.g. second row on attached image.
<img width="1221" alt="Screenshot 2023-05-16 at 09 10 03" src="https://github.com/CodeForAfrica/ui/assets/1779590/2e0f51de-1401-4deb-acb2-9a6bbcd50e2c">

Reply:
@kilemensi On my end I don't have the padding. Perhaps it's a space from the CMS? @koechkevin How does yours look like?
![image](https://github.com/CodeForAfrica/ui/assets/43873157/c534873a-d801-4039-ae72-562c2301bef8)

Reply:
Duude focus. Who puts spaces in the CMS?

No there are no space(s) in the CMS
<img width="416" alt="Screenshot 2023-05-16 at 12 25 54" src="https://github.com/CodeForAfrica/ui/assets/1779590/05e27ba0-efde-4a58-a9fc-e8c67b8a804c">

Yes, it's styling issue. By default a button has `min-size`. Yours doesn't show probably because the text you're using is more than `min-size` i.e. `64px`. Mine shows because it's less than that.
<img width="1240" alt="Screenshot 2023-05-16 at 12 25 31" src="https://github.com/CodeForAfrica/ui/assets/1779590/bef3d65e-34b3-4a51-8f6f-5b97c74dfcab">